__tests__/app.test.js

```diff
@@ -217,5 +217,65 @@ describe('checks specific articles comments',() =>{
 })
 
 describe('posting a new comment to a specific article',() =>{
+    test('testing it returns a 201 status',() =>{
+        const myPost = {
+            username : "butter_bridge",
+            body : "Great article"
+        }
+        return request(app)
+        .post('/api/articles/4/comments').send(myPost)
+        .expect(201)
+        .then((result) =>{
+            expect(result._body.comment_id).toBe(19)
+            expect(result._body.body).toBe('Great article')
+            expect(result._body.article_id).toBe(4)
+            expect(result._body.author).toBe('butter_bridge')
+            expect(result._body.votes).toBe(0)
+            expect(typeof(result._body.created_at)).toBe('string')
+        })
+    })
+    test('Test for username who does not exist',() =>{
+        const myPost = {
+            username : "dan",
+            body : "Great article"
+        }
+        return request(app)
+        .post('/api/articles/4/comments').send(myPost)
+        .expect(404).then((response) =>{
+        expect(response.body.msg).toBe('The name you gave is not a current user')
+    })
+    })
+    test('Tests for an invalid comment',() =>{
```

Comment:
I would rename this, as the comment itself here is valid, you are posting using an invalid id